pkg/portal/portal.go

```diff
@@ -137,12 +141,12 @@ func (p *portal) setupRouter() error {
 	r := mux.NewRouter()
 	r.Use(middleware.Panic(p.log))
 
-	assetv1, err := Asset("v1/build/index.html")
+	assetv1, err := assets.EmbeddedFiles.ReadFile("v1/build/index.html")
 	if err != nil {
 		return err
 	}
 
-	assetv2, err := Asset("v2/build/index.html")
+	assetv2, err := assets.EmbeddedFiles.ReadFile("v2/build/index.html")
 	if err != nil {
 		return err
 	}
```

Comment:
I think I'd call that `indexV{1,2}` here.  Not your change but the name didn't really make sense before so we might as well change it